composeApp/build.gradle.kts

```diff
@@ -0,0 +1,124 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsCompose)
+    alias(libs.plugins.compose.compiler)
+    kotlin("plugin.serialization") version "1.5.30"
+}
+
+kotlin {
+    androidTarget {
+        @OptIn(ExperimentalKotlinGradlePluginApi::class)
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_11)
+        }
+    }
+    
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach { iosTarget ->
+        iosTarget.binaries.framework {
+            baseName = "ComposeApp"
+            isStatic = true
+        }
+    }
+    
+    sourceSets {
+        
+        androidMain.dependencies {
+            implementation(compose.preview)
+            implementation(libs.androidx.activity.compose)
+            implementation(libs.koin.android)
+            implementation(libs.koin.compose)
+            implementation(libs.koin.androidx.compose)
+        }
+        commonMain.dependencies {
+            implementation(libs.kermit)
+            implementation(libs.landscapist.coil3)
+
+            api(libs.koin.core)
+            implementation(libs.koin.compose)
+            implementation(libs.navigator.koin)
+            implementation(libs.navigator.screen.model)
+
+            // Ktor for networking
+            implementation(libs.ktor.client.core)
+            implementation(libs.ktor.client.cio) // or another engine
+            implementation(libs.ktor.client.content.negotiation)
+            implementation(libs.ktor.serialization.kotlinx.json)
+            implementation(libs.kotlinx.serialization.json)
+            implementation(libs.ktor.serialization.kotlinx.json)
+
+//            implementation(libs.kstatemachine)
+//            implementation(libs.kstatemachine.coroutines)
+            val voyagerVersion = "1.1.0-beta02"
+
+            implementation("cafe.adriel.voyager:voyager-screenmodel:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-bottom-sheet-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-tab-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-transitions:$voyagerVersion")
+            //implementation("cafe.adriel.voyager:voyager-lifecycle-kmp:$voyagerVersion")
+//            implementation("media.kamel:kamel-image:0.9.5")
```

Comment:
could you clean dependency list? 
some of them seems to be useless here, like ktor for instance
I also don't know the purpose of kermit, coil3 and voyager libraries.